### ISSUE TITLE
Update BluePic-Server to Swift 3

### ISSUE THREAD

**Issue by king33** [IBM-Swift/BluePic#312]

**Comment by rfdickerson** [comment]
Does this have a corresponding PR?

**Comment by rfdickerson** [comment]
What's the plan regarding using both Facebook Credentials and MCA Credentials when this project is merged into develop?

**Comment by king33** [comment]
#314

**Comment by king33** [comment]
Here's my plan for BluePic-web:

1. Upgrade BluePic-Server, Watson SDK and iOS app and test functionality
2. Merge in the web project with facebook login disabled.  Authentication will basically be disabled at this point and the BluePic-web profile page will not be available. 
3. Address the MCA issue of supporting JS clients in order to enable MCA/ FB auth in the web project.

Let me know your thoughts, @rfdickerson

**Comment by rfdickerson** [comment]
Yeah, I think leave that line(s) of code in the project around FacebookCredentials but inside of an `if-else` block. At the top of the project, have a boolean variable that can be set that would toggle between MCA or Facebook Auth. We can document how to enable Facebook login by changing that variable's value in the README or something.

**Comment by king33** [comment]
Another short-term fix may be to simply try to register the FB plugin first.  I believe Schmuel said that if FB creds aren't found, a 'Pass' is returned and then the next plugin (MCA) would be tried.  We tested registering MCA first, but apparently it doesn't return a 'Pass' when it fails.  Did I misunderstand?  If that's the case, it is a hacky fix, but would work until we've fixed the credentials system.
